docs/DeveloperGuide.adoc

```diff
@@ -297,33 +297,47 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new patient/donor/doctor |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |delete a patient/donor/doctor |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |find a patient/donor/doctor by certain keywords |locate details of persons without having to go through the entire list
```

Comment:
`... doctor by certain keywords ...` change to `... doctor using certain keywords ... `